### ISSUE TITLE
Proposal: continuous text output from script

### ISSUE THREAD

**Issue by rmirabelli** [JohnSundell/Marathon#134]
Currently Marathon prints all of the script's output at the end of the script running.

I'd love to see Marathon instead use a Pipe() along with waitForDataInBackgroundAndNotify(), rather than use a print statement at script completion. This could probably be done by creating a different printer that is passed along to shellOut.

I'm happy to work on this request myself, but would appreciate input from others as to potential impacts, especially on Linux.

**Comment by jarod2d** [comment]
Yes please! I was just struggling for a while to figure out why I couldn't get output in real time. In the mean time, is there any workaround without ditching Marathon?

**Comment by rmirabelli** [comment]
I don't know of a workaround, but I'm going to start working on this over the next day or two, and it shouldn't be too challenging.

**Comment by JohnSundell** [comment]
I'd really love to see this happen 👍 It will require adding async/continuous output to ShellOut, here's an issue for that: https://github.com/JohnSundell/ShellOut/issues/11
